### PR TITLE
Ports Thanos Car from AuStation Again

### PR DESCRIPTION
**Pull request by MCterra10**
## About The Pull Request
Unfortunately my last PR got snapped into oblivion, so I've made another one as this has to be the half that survives the merge

## Why It's Good For The Game
Fun new (dead) meme content for admins

## Changelog
:cl:
add: THANOS CAR! THANOS CAR!
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
bad pr dont merge this

**Comment by MCterra10** [comment]
@carlarctg it's only funny when you do it on your own PRs

**Comment by wesoda25** [comment]
bad pr dont merge this

**Comment by MCterra10** [comment]
Orang you can literally look at the old PR for pictures
#48244

**Comment by MMMiracles** [comment]
you also could of just posted the picture from the PR you linked instead with about as much effort

**Comment by MCterra10** [comment]
@MMMiracles can't upload pictures to github with my phone m9

**Comment by MCterra10** [comment]
Perhaps you could be a good Samaritan and upload them for me

**Comment by MCterra10** [comment]
God damn it, can we add a confirmation to the close and comment button please

**Comment by HomocidlePancakes** [comment]
@mcterra You can upload pics from phone to github. Switch to desktop version

File: code/modules/vehicles/cars/thanoscar.dm [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	max_integrity = 250
+	armor = list("melee" = 70, "bullet" = 40, "laser" = 40, "energy" = 0, "bomb" = 30, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 80)
+	enter_delay = 20
+	max_occupants = 50
+	movedelay = 0.6
+	key_type = null
+
+/obj/vehicle/sealed/car/thanoscar/Bump(atom/movable/M)
+	. = ..()
+	if(isliving(M))
```
Comment (GuyonBroadway) [inline code/modules/vehicles/cars/thanoscar.dm]: If(prob(50))
    M.Dust()

**Comment by AnturK** [comment]
Didn't i close this already ?

**Comment by Tharcoonvagh** [comment]
Noo

**Comment by trollbreeder** [comment]
PSA: Dont be like goof when your meme PR dies and gets closed, KEEP THAT SHIT DEAD.

**Comment by MCterra10** [comment]
This is so sad guys, Alexa play despacito

**Pull request closed without merging**